### ISSUE TITLE
Xfce4-gtk3: Appearance from settings can only change theme at the 1st user and not the others

### ISSUE THREAD

**Issue by ThanosApostolou** [manjaro/packages-extra#129]
Reproduced with my custom ISO which uses xfce4-gtk3 and with the latest manjaro-xfce-sx-17.1-pre2.

Steps to reproduce:
1. Install xfce4-gtk3 group
2. Use msm user utility to create a new user.
3. Login with the new user and try to change **Style** theme and verify that it's using Adwaita no matter what you set.
4. Login with your original user and verify that Appearance it's working properly

I don't know if it's to blame **xfconf-gtk3** or **xfce4-settings-gtk3** or something else and I have no idea how to debug this.

**Comment by philmmanjaro** [comment]
Yes, it is a known issue. We had [patched](https://github.com/manjaro/packages-extra/blob/master/xfce4-gtk3/xfce4-settings-gtk3/PKGBUILD) **xfsettingsd** not to be run as a daemon for now. We had partly issues with the first user as well on our **Spitfire**.

**Comment by ThanosApostolou** [comment]
Hmmm... I built **xfce4-settings-gtk3** without the patch and the workaround and the problem solved.  It seems to run fine on my end. I haven't experienced any crashes yet,  maybe we can found any better workaround?

**Comment by philmmanjaro** [comment]
It is an issue on the Spitfire for which we have not yet a proper solution found.
